doc/plutus-core-spec/flat-serialisation.tex

```diff
@@ -559,26 +524,7 @@ \subsection{Built-in types}
 \subsection{Constants}
 \label{sec:flat-constants}
 Values of built-in types can mostly be encoded quite simply by using encoders
-already defined.  Note that the unit value \texttt{(con unit ())} does not have
```

Comment:
I moved these commenst to notes after the decoder to make them a bit more obvious.